akka-docs-dev/rst/java/stream-graphs.rst

```diff
@@ -194,6 +194,24 @@ together and also turned around with the ``.reversed()`` method. The test
 simulates both parties of a network communication protocol without actually
 having to open a network connection—the flows can just be connected directly.
 
+.. _graph-matvalue-java:
+
+Accessing the materialized value inside the Graph
+-------------------------------------------------
+
+In certain cases it might be necessary to feed back the materialized value of a Graph (partial, closed or backing a
+Source, Sink, Flow or BidiFlow). This is possible by using ``builder.matValue`` which gives an ``Outlet`` that
```

Comment:
I am also not particularly happy with "mat" things but so far has found no better solution. I created a ticket: https://github.com/akka/akka/issues/17123